.github/workflows/kind.yml

```diff
@@ -488,6 +488,8 @@ jobs:
       run: |
         docker load -i antrea-ubuntu.tar
         docker tag antrea/antrea-ubuntu-coverage:latest antrea/antrea-ubuntu:latest
```

Comment:
You can remove this if above step skip saving it.

Reply:
okay